README.md

```diff
@@ -31,6 +31,8 @@ These instructions will get you a copy of the project up and running on your loc
 - A running instance of Saleor.
 
     To run the storefront, you have to set the `BACKEND_URL` environment to point to the Saleor instance. If you are running Saleor locally with the default settings, set `BACKEND_URL` to: `http://localhost:8000/`.
+    <br>**For Quick Demo** use `BACKEND_URL` = `https://demo.getsaleor.com` <br>
+    **Or** `https://demo.getsaleor.com/graphql` for `API_URL` in src/index.tsx
```

Comment:
I feel like we shouldn't tell the user to change the `API_URL` constant as it's not a proper way to proceed. Talking about changing the source code is too low level for the user for this kind of documentation. This documentation should only be for high level interfaces.

Reply:
I agree about not mentioning `API_URL` change, could we remove that? We could mention though that people may try the storefront with the public demo as a backend, but I'd rephrase that a little. How about:

```
You can also use the public demo instance as a backend, but it runs in read-only mode and some features may be unavailable. To use it, set `BACKEND_URL` to `https://demo.getsaleor.com/`.
```